modules/gcve-private-cloud/variables.tf

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "description" {
+  description = "Private cloud description."
+  type        = string
+  default     = "Terraform-managed."
+}
+
+variable "management_cidr" {
+  description = "vSphere/vSAN subnets CIDR range."
+  type        = string
```

Comment:
is there a sensible default that can be provided here?

Reply:
here I would recommend to not add any default and force the user to set a proper value. This CIDR needs to be designed according to the expected scalability of the environment and any change will require the cluster destruction and re-creation.

Reply:
I would then note this in the variable description.